app/Http/Controllers/MasjidProfileController.php

```diff
@@ -39,4 +40,32 @@ public function update(Request $request): RedirectResponse
 
         return redirect()->route('masjid_profile.show');
     }
+
+    public function image(Request $request)
+    {
+        $this->authorize('edit_masjid_profile');
+
+        $validatedPayload = $request->validate([
+            'image' => 'required'
+        ]);
+
+        if (!base64_decode($validatedPayload['image'])) {
+            return response()->json([
+                'message' => 'Image not found'
```

Comment:
Pakai lang, supaya kedepannya bisa multi bahasa.
```suggestion
                'message' => __('masjid_profile.image_not_found'),
```
Jangan lupa tambahkan lang di `resources/lang/(en|id)/masjid_profile.php`